src/parsers/perf/perfparser.cpp

```diff
@@ -1055,9 +1055,10 @@ class PerfParserPrivate : public QObject
         for (const auto& cost : sample.costs) {
             if (core.costs.size() <= cost.attributeId) {
                 const auto oldSize = core.costs.size();
-                core.costs.resize(cost.attributeId + 1);
+                const auto newSize = cost.attributeId + 1;
```

Comment:
this part isn't needed anymore, is it?

Reply:
it was never needed, just useful, and that's still the case as we don't need to compute the newSize in the loop